assembl/static2/js/app/components/debate/thread/nuggets.jsx

```diff
@@ -45,15 +46,59 @@ class Nuggets extends React.Component {
   }
 
   render() {
-    const { extracts } = this.props;
+    const { extracts, isHarvesting } = this.props;
     const { top } = this.state;
+    if (isHarvesting) {
+      if (Array.isArray(extracts) && extracts.length > 0) {
+        return (
+          <div
+            ref={(node) => {
+              this.node = node;
+            }}
+            className="extracts extracts--is-harvesting"
+            style={Nuggets.topToStyle(top)}
+          >
+            {extracts.map(extract => (
+              <div key={extract.id} className="extract">
+                <div className="extract-header">
+                  <div className="extract-harvester">
+                    <div className="extract-harvester-left">
+                      <div className="extract-harvester-avatar">
+                        <img
+                          src="/static2/img/icons/avatar.png?TODO"
+                          width="30"
+                          height="30"
+                          alt="Albert Boyd TODO"
+                          onError={(e) => {
+                            const target = e.target;
+                            target.src = '/static2/img/icons/avatar.png';
+                          }}
+                        />
+                      </div>
+                    </div>
+                    <div className="extract-harvester-right">
+                      <div className="extract-harvester-name">Albert Boyd TODO</div>
+                      <div className="extract-harvesting-date">2 days ago TODO</div>
```

Comment:
To be changed later...

Reply:
same remark as https://github.com/assembl/assembl/pull/352#discussion_r173413452